landoui/static/src/js/pages/RevisionPage.js

```diff
@@ -0,0 +1,33 @@
+/* Simple class to manage events and state on the /revision/{id} page */
+Lando.RevisionPage = class {
```

Comment:
In looking at how this is used, we should really use the jQuery plugin pattern.  You can find an example I created for MDN here:

https://github.com/mozilla/kuma/blob/master/kuma/static/js/moz-jquery-plugins.js#L287